test/Dialect/FIRRTL/errors.mlir

```diff
@@ -1746,3 +1746,13 @@ firrtl.circuit "OpenBundleNotFieldID" {
   // expected-error @below {{bundle element "a" has unsupported type that does not support fieldID's: '!firrtl.string'}}
   firrtl.extmodule @OpenBundleNotFieldID(out out : !firrtl.openbundle<a: string>)
 }
+
+// -----
+// Strict connet between non-equivalent anonymous type operands.
```

Comment:
```suggestion
// Strict connect between non-equivalent anonymous type operands.
```
(small typo, although phrasing here doesn't match language used elsewhere in this PR, shrug)